staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/typechecking.go

```diff
@@ -268,7 +268,16 @@ func (c *TypeChecker) typesToCheck(p *v1beta1.ValidatingAdmissionPolicy) []schem
 					}
 					resolved, err := c.RestMapper.KindsFor(gvr)
 					if err != nil {
-						continue
+						if restMapperRefreshAttempted {
+							// RESTMapper refresh happens at most once per policy
```

Comment:
It's actually once per policy per reconcile, right?

Reply:
Yes! Because there is no schema-watching re-requeue, a policy can be rechecked only due to a mutation to itself. If the policy mutated, the referred set of resources may change too (and we don't track that), thus once per policy-reconcile is reasonable.